pkg/util/strings.go

```diff
@@ -0,0 +1,10 @@
+package util
```

Comment:
This method almost verbatim is in wrangler already if you want to use it: https://github.com/rancher/wrangler/blob/master/pkg/slice/contains.go#L3